test/test_grade/test_integration.py

```diff
@@ -246,6 +275,7 @@ def test_config_overrides(mocked_launch_grade):
         "percent_correct": float('nan'),
         "total_points_earned": 15.0,
         "file": POINTS_POSSIBLE_LABEL,
+        "grading_status": "--"
```

Comment:
Why is this grading status `--`? I think we shouldn't ever have a status of `--`. It should always be a string that indicates the status -- either completed or errored.

Reply:
The is on the points-per-question row -- I changed it to "NA".

Reply:
The "NA" gets coerced by pandas' read_csv in the testing. I am suggesting for `grading_status` on the points-per-question row we use the string "--".